src/main/java/uk/gov/hmcts/darts/common/exceptions/soap/documentum/ServiceExceptionType.java

```diff
@@ -53,23 +59,26 @@ public ServiceExceptionType(String code, Throwable cause, String arg) {
 
         DfsAttributeHolder messageIdAttribute = new DfsAttributeHolder();
         messageIdAttribute.setName(ATTRIBUTE_MESSAGE_ID);
-        messageIdAttribute.setType(getMessage().getClass().getCanonicalName());
-        messageIdAttribute.setValue(getMessageId());
+        messageIdAttribute.setType(message.getClass().getCanonicalName());
+        messageIdAttribute.setValue(messageId);
 
         // add one exception
         DfsExceptionHolder holder = new DfsExceptionHolder();
-        holder.setExceptionClass(DOCUMENTUM_SERVICE_CONTEXT_EXCEPTION_NAME);
+        holder.setExceptionClass(cause.getClass().getCanonicalName());
```

Comment:
Question here as to whether we should stop hard coding this? I would prefer to return an actual exception class